### PR TITLE
AIS-64 | @jdwjdwjdw | A11y and console fixups

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# NOT READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- [AIS-64](https://stanfordits.atlassian.net/browse/AIS-64): A11y | Decanter: Fix WCAG A Issues
  - `Form field missing a label` A11y issues at https://decanter.stanford.edu/examples/form-elements
  - Fixups for some other issues noted by the Console

# Review By (Date)
- Before 6/1

# Criticality
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Download the Siteimprove Accessibility Checker extension at https://chromewebstore.google.com/detail/siteimprove-accessibility/djcglbmbegflehmbfleechkjhmedcopn
3. Navigate to `/examples/form-elements`, run the Sitemprove extension, and confirm that the `Form field missing a label` A11y issues are no longer being flagged on that page.
4. Review other updated pages and confirm they still look good.
5. Review code 🐙 

# Associated Issues and/or People
- [AIS-64](https://stanfordits.atlassian.net/browse/AIS-64): A11y | Decanter: Fix WCAG A Issues

[AIS-64]: https://stanfordits.atlassian.net/browse/AIS-64?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AIS-64]: https://stanfordits.atlassian.net/browse/AIS-64?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/examples/heros.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ lastUpdatedDate: 2022-04-06T12:00:00.000Z
         <div class="h-full w-full overflow-hidden absolute">
           <img
             class="h-full w-full object-cover object-center"
-            src="https://placekitten.com/1000/750"
```
Comment (sherakama) [inline content/examples/heros.mdx:12]: Ah, boo. No more place kitteh.

File: content/examples/cards.mdx [hunk 2]
```diff
@@ -200,29 +200,29 @@ lastUpdatedDate: 2023-05-25T12:00:00.000Z
   </Section>
   <Section heading="Library Card" width="full">
   ```html
-  <div class="card block w-full basefont-23 leading-display bg-white text-black border border-solid border-black-10 shadow-md md:max-w-500">
-    <div class="overflow-hidden aspect-[16/9] relative" aria-hidden="true">
+  <div className="card block w-full basefont-23 leading-display bg-white text-black border border-solid border-black-10 shadow-md md:max-w-500">
```
Comment (sherakama) [inline content/examples/cards.mdx:203]: @jdwjdwjdw 
It looks like you've changed the example markup from HTML valid markup to React markup. Can you tell me more about why you changed these? This changes the example markup code and they are invalid HTML attributes. 

Screenshot 2024-05-29 at 09 43 02

Comment (sherakama) [reply content/examples/cards.mdx:203]: The stuff inside the html fencing `\`\`\`html` should stay as html and not react.

**Review by sherakama — CHANGES_REQUESTED** [review]
Thanks for adding the labels. 

For the HTML code inside the code fence, please return it to HTML markup. 

Otherwise this all looks good.

File: content/examples/heros.mdx (hunk 1 above)
Comment (jdwjdwjdw) [reply content/examples/heros.mdx:12]: I was sad too 😿

File: content/examples/cards.mdx (hunk 2 above)
Comment (jdwjdwjdw) [reply content/examples/cards.mdx]: Thanks - I got a little overambitious while trying to fix the console errors. Some of the JSX vs HTML did need a little cleanup, so I removed my incorrect changes and kept the ones that needed updating. Should be good for another look now.

**Comment by sherakama** [comment]
Screenshot 2024-05-29 at 10 57 32

**Pull request merged**